content/security/docs/workloadsecurity.md

```diff
@@ -0,0 +1,56 @@
+# Application Workload Security
+
+It is important to secure your application workload communications. A main concern is encryption in transit so that your applications and Kubernetes pods can securely communicate with each other. Networking between applications should be secured using TLS.
```

Comment:
There is a section in networking security where we talk about TLS. Is there a reason you created a separate category for workload security?

Reply:
We created a separate category following the guidance provided by Micah Hausler during a review of the content.

Reply:
First, thanks for this PR @dcamzn and @divyansh-gupta. I would prefer to move this to the section on [network security/encryption](https://aws.github.io/aws-eks-best-practices/security/docs/network/#encryption-in-transit_1) in transit for now.